packages/angular/src/utils/cli-config-utils.ts

```diff
@@ -0,0 +1,50 @@
+export function replaceAppNameWithPath(
+  node: string | string[] | object | unknown,
+  appName: string,
+  root: string
+): any {
+  if (typeof node === 'string') {
+    const matchPattern = new RegExp(
+      `([^a-z0-9]*(${appName}))|((${appName})[^a-z0-9:]*)`,
+      'gi'
+    );
+    if (
+      !!node.match(matchPattern) &&
+      node !== 'application' &&
+      node !== 'library'
+    ) {
+      const r = node.replace(appName, root);
+      return r.startsWith('/apps') || r.startsWith('/libs')
+        ? r.substring(1)
+        : r;
+    } else {
+      return node;
+    }
+  } else if (Array.isArray(node)) {
+    return node.map((j) => replaceAppNameWithPath(j, appName, root));
+  } else if (typeof node === 'object' && node) {
+    const forbiddenPropertyList: string[] = [
+      'prefix',
+      'builder',
+      'executor',
+      'browserTarget',
```

Comment:
I know this is a copy and paste from existing functionality and not an issue introduced by this PR, but we also have a `buildTarget` property in some of our builders that is similar to `browserTarget`. We should probably include it in this list as well.